pdc/apps/release/serializers.py

```diff
@@ -205,3 +207,33 @@ class VariantTypeSerializer(StrictSerializerMixin, serializers.ModelSerializer):
     class Meta:
         model = VariantType
         fields = ('name',)
+
+
+class ReleaseGroupSerializer(StrictSerializerMixin, serializers.ModelSerializer):
+    description         = serializers.CharField(required=True)
+    type                = serializers.SlugRelatedField(slug_field='name',
+                                                       queryset=ReleaseGroupType.objects.all())
+    releases            = ChoiceSlugField(slug_field='release_id',
+                                          many=True, queryset=Release.objects.all(),
+                                          allow_null=True, required=False)
+    active              = serializers.BooleanField(default=True)
+
+    class Meta:
+        model = ReleaseGroup
+        fields = ('name', 'description', 'type', 'releases', 'active')
+
+    def to_internal_value(self, data):
+        releases = data.get('releases', [])
+        for release in releases:
+            try:
+                Release.objects.get(release_id=release)
+            except Release.DoesNotExist:
+                raise serializers.ValidationError({'detail': '%s release does not exist' % release})
+
+        type_key = data.get('type', None)
+        if type_key:
+            try:
+                ReleaseGroupType.objects.get(name=type_key)
+            except ReleaseGroupType.DoesNotExist:
+                raise serializers.ValidationError({'detail': '%s type does not exist' % type_key})
```

Comment:
I think type you should use it's default behavior. It's better to show the legal value. Maybe you could implement `to_internal_value` in field level. Here maybe release field.